routing/route.cpp

```diff
@@ -113,21 +113,32 @@ double Route::GetCurrentTimeToEndSec() const
 
   double const curSegSpeedMPerS = curSegLenMeters / curSegTimeS;
   CHECK_GREATER(curSegSpeedMPerS, 0, ("Route can't contain segments with zero speed."));
+  /// @todo GetDistFromCurPointTo!Next!RoutePointMeters should be used for calculalation of remaining segment length.
   return totalTimeS - (etaToLastPassedPointS +
                        m_poly.GetDistFromCurPointToRoutePointMeters() / curSegSpeedMPerS);
 }
 
+void Route::GetCurrentSpeedLimit(SpeedInUnits & speedLimit) const
```

Comment:
Why not returning/checking the value? What if IsValid() == false?

Reply:
OK